docs/website/docs/tutorial/filesystem.md

```diff
@@ -0,0 +1,276 @@
+```
+  </TabItem>
+
+<TabItem value="code">
+```py
+import os
+
+from dlt.common.configuration.specs.aws_credentials import AwsCredentials
+from dlt.sources.filesystem import filesystem
+
+parquet_resource = filesystem(
+    bucket_url="s3://<bucket_name>/<path_to_files>/",
+    # please, do not specify secrets values directly in the code,
+    # instead, you can use env variables to get the credentials
+    credentials=AwsCredentials(
+        aws_access_key_id=os.environ["AWS_KEY_ID"],
+        aws_secret_access_key=os.environ["AWS_SECRET"],
+    )
+    file_glob="**/*.parquet") | read_parquet()
+```
+</TabItem>
+</Tabs>
+
+As you can see all parameters of `filesystem` can be specified in the code or taken from the configuration.
+
+:::tip
+`dlt` supports more ways of authorizing with the cloud storage, including identity-based
+and default credentials. To learn more about adding credentials to your pipeline, please refer to the
+[Configuration and secrets section](../../../general-usage/credentials/complex_types#aws-credentials).
+:::
+
+## 4. Running the pipeline
+
+Let's verify that the pipeline is working as expected. Run the following command to execute the pipeline:
+
+```sh
+python filesystem_pipeline.py
+```
+
+You should see the output of the pipeline execution in the terminal. The output will also display the location of the DuckDB database file where the data is stored:
+
+```sh
+Pipeline s3_to_duckdb load step completed in 1.08 seconds
+1 load package(s) were loaded to destination duckdb and into dataset parquet_data
+The duckdb destination used duckdb:////home/user-name/quick_start/s3_to_duckdb.duckdb location to store data
+Load package 1692364844.3523829 is LOADED and contains no failed jobs
+```
+
+## 5. Exploring the data
+
+Now that the pipeline has run successfully, let's explore the data loaded into DuckDB. `dlt` comes with a built-in browser application that allows you to interact with the data. To enable it, run the following command:
+
+```sh
+pip install streamlit
+```
+
+Next, run the following command to start the data browser:
+
+```sh
+dlt pipeline s3_to_duckdb show
+```
+
+The command opens a new browser window with the data browser application. `s3_to_duckdb` is the name of the pipeline defined in the `filesystem_pipeline.py` file.
+You can explore the loaded data, run queries and see some pipeline execution details.
+
+## 6. Appending, replacing, and merging loaded data
+
+Try running the pipeline again with `python filesystem_pipeline.py`. You will notice that
+all the tables have data duplicated. This happens because by default, `dlt` appends the data to the destination table. It is very useful, for example, when you have daily data updates and you want to ingest them. In `dlt` you can control how the data is loaded into the destination table by setting the `write_disposition` parameter in the resource configuration. The possible values are:
```

Comment:
```suggestion
all the tables have duplicate data. This happens because by default, `dlt` appends the data to the destination table. It is very useful, for example, when you have daily data updates and you want to ingest them. In `dlt` you can control how the data is loaded into the destination table by setting the `write_disposition` parameter in the resource configuration. The possible values are:
```

Reply:
I left some comments about this paragraph on Anton's PR as well -  we should coordinate that.